src/org/traccar/sms/HttpSmsClient.java

```diff
@@ -78,12 +81,13 @@ private String preparePayload(String destAddress, String message) {
 
     private Invocation.Builder getRequestBuilder() {
         return Context.getClient().target(url).request()
-                .header(SecurityRequestFilter.AUTHORIZATION_HEADER, authorization);
+                .header(authorizationName, authorization);
     }
 
     @Override
     public void sendMessageSync(String destAddress, String message, boolean command) throws MessageException {
-        Response response = getRequestBuilder().post(Entity.entity(preparePayload(destAddress, message), mediaType));
+        Response response = getRequestBuilder().post(Entity.entity(
+                preparePayload(destAddress, message.trim()), mediaType));
```

Comment:
I think we should do trim in the `preparePayload` method.

Reply:
Done.